CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ### 1.6.1 (Next)
+* [#111](https://github.com/dblock/iex-ruby-client/pull/111): Added support for fetching symbols for a region and updates ref_data_symbols_for_exchange according to API change - [@mathu97](https://github.com/mathu97).
```

Comment:
Split in 2 separate lines, even if it's the same PR. The update to `ref_data_symbols_for_exchange` should say "Added Symbols#exchange_suffix, exchange_name, ...".